.gitignore

```
@@ -39,11 +26,13 @@ vignettes/*.pdf
 # R Environment Variables
 .Renviron
 
-# pkgdown site
-docs/
+# Data
+_targets/
 
```

```suggestion
data/
```

I know this is implied when you ignore the .feather files, but maybe nice to do anyway? 100% your call.